generators/app/index.js

```diff
@@ -41,6 +41,7 @@ var isGoWeb = false;
 var baseImageName = '';
 var updateProjectJsonNoteForUser = null;
 var updateProgramCSNoteForUser = null;
+var isDotNetWeb = true;
```

Comment:
Is this being logged in the telemetry?

---
In reply to: [71039209](https://github.com/Microsoft/generator-docker/pull/103#discussion_r71039209) [](ancestors = 71039209)